src/app/researcher/study/[studyId]/review/page.tsx

```diff
@@ -1,23 +1,26 @@
-import { Center, Group, Paper, Stack, Title } from '@mantine/core'
+import { Center, Group, Paper, Stack } from '@mantine/core'
 import { db } from '@/database'
 import { AlertNotFound } from '@/components/errors'
 import { ResearcherBreadcrumbs } from '@/components/page-breadcrumbs'
+import { checkUserAllowedStudyView, latestJobForStudy } from '@/server/db/queries'
+import { ViewCSV } from './results'
 
 export default async function StudyReviewPage(props: { params: Promise<{ studyId: string }> }) {
-    const params = await props.params
+    const { studyId } = await props.params
 
-    const { studyId } = params
+    await checkUserAllowedStudyView(studyId)
 
-    // TODO check user permissions
     const study = await db.selectFrom('study').selectAll().where('id', '=', studyId).executeTakeFirst()
 
+    const job = await latestJobForStudy(studyId)
+
     if (!study) {
         return <AlertNotFound title="Study was not found" message="no such study exists" />
     }
 
     return (
         <Center>
-            <Paper w="70%" shadow="xs" p="sm" m="xs">
+            <Paper shadow="xs" p="xl" w="100%">
```

Comment:
we don't need the `<Center>` and the subsequent w="100%" on the `Paper` element, im sure there will be a round of hi-fi work to do on these components so its fine to leave for now too

Reply:
hah, nope we do not, I'll remove Center